.github/workflows/build_dev_docs.yml

```diff
@@ -0,0 +1,31 @@
+---
+name: Deploy Development Documentation
+on:
+  push:
+    paths:
+      - 'docs/**'
+      - mkdocs.yml
+    branches:
+      - main
+jobs:
+  deploy:
+    name: Deploy Development Documentation
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout main
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          persist-credentials: true
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.x
+      - name: Install Dependencies
+        run: |
+          pip install -r docs/requirements.txt
+      - name: Configure Git User
+        run: |
+          git config user.name "ci-docs"
+          git config user.email "ci-docs@gmail.com"
+      - name: Deploy Development Docs
+        run: mike deploy --push dev
```

Comment:
what is dev?

Reply:
Wrote in the description: dev contains current status of docs in our main

Reply:
Can we not just call it `main`?

Reply:
For me it makes more sense to call it dev there since main might be interpreted as the main documentation and not that it is what we have on main and is currently in development until next release. But I see it confused you and @tamirkamara  so I'll change it to main .

Reply:
main? What will a regular user think that is supposed to refer to?
Sorry but it's is still confusing

Reply:
decided on "unreleased" and "latest-release"